app/services/mentions/create_all.rb

```diff
@@ -10,32 +10,55 @@ def self.call(*args)
 
     def call
       # Only works for comments right now.
+      mentioned_users = users_mentioned_in_notifiable_text_that_are_existing_and_not_its_owners
```

Comment:
I wonder if `mentioned_usernames` and `mentioned_users` could just become memoized instance variables that are set the first time that they are called, and then reused in the service whenever necessary? Something like:

```ruby
def mentioned users
  @mentioned_users ||= ...
end
```

Reply:
As I see, currently, `Mentions::CreateAll` is only created (and `call` called) once inside a worker, so memoization won't do much for now.

Reply:
@vaidehijoshi I think line 13 is revealing where mentioned users are coming from, so I like it that way.